keras/layers/convolutional.py

```diff
@@ -1449,7 +1464,17 @@ class ZeroPadding2D(Layer):
     # Arguments
         padding: tuple of int (length 2)
             How many zeros to add at the beginning and end of
-            the 2 padding dimensions (axis 3 and 4).
+            the 2 padding dimensions (rows and cols) symmetrically.
+            Setting padding to the values different from (1, 1)
+            overrides top_pad, bottom_pad; and left_pad, right_pad arguments.
```

Comment:
@fchollet It was a polymorphic initially, but you asked me to come up with another solution ;)
No problem, I think I'll go for a tuple of length 2 and a dictionary.